core/src/main/scala/scalaz/Void.scala

```diff
@@ -0,0 +1,29 @@
+package scalaz
+
+import Liskov.<~<
+
+trait VoidModule {
+  type Void
+
+  def absurd[A](v: Void): A
+
+  def unsafeVoid: Void
```

Comment:
What do we have `unsafeVoid` for, again?

Reply:
honestly, I don't know.. I copied it from @alexknvl's implementation. So, he might have better idea. :)

Reply:
For example, to convince the compiler that `(Int === String) => Void`.